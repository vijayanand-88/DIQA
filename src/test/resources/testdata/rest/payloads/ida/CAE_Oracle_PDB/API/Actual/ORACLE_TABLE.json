[
  {
    "sample": {
      "header": [
        "ID",
        "NAME",
        "DOB",
        "DOB2",
        "DOB3",
        "DOB4"
      ],
      "values": [
        [
          "200",
          "Test Name2",
          "2005-10-20 00:00:00.0",
          "2005-05-01 06:14:00.0",
          "2005-05-01 15:14:00.742",
          "2005-07-01 18:14:00.742"
        ],
        [
          "100",
          "Test Name1",
          "2015-11-10 00:00:00.0",
          "2015-07-01 02:11:00.0",
          "2015-01-31 22:44:00.742",
          "2015-07-01 05:45:00.742"
        ],
        [
          "300",
          "Test Name3",
          "1978-10-10 00:00:00.0",
          "1978-07-01 11:11:00.0",
          "1978-02-01 06:44:00.742",
          "1978-07-01 00:45:00.742"
        ],
        [
          "400",
          "Test Name4",
          "1945-01-01 00:00:00.0",
          "1945-01-01 11:11:00.0",
          "1945-01-01 05:44:00.742",
          "1944-12-31 22:45:00.742"
        ],
        [
          "500",
          "Test Name5",
          "2000-12-01 00:00:00.0",
          "2000-12-01 11:11:00.0",
          "2000-12-01 06:44:00.742",
          "2000-11-30 23:45:00.742"
        ]
      ]
    },
    "id": "null"
  }
]