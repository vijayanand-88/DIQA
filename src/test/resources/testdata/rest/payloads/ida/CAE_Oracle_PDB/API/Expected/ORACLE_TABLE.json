[{"id":"null","sample":{"values":[["","","","","2005-05-01 13:14:00.742","2005-07-01 16:14:00.742"],["","","","","2015-01-31 21:44:00.742","2015-07-01 03:45:00.742"],["","","","","1978-02-01 05:44:00.742","1978-06-30 22:45:00.742"],["","","","","1945-01-01 04:44:00.742","1944-12-31 21:45:00.742"],["","","","","2000-12-01 05:44:00.742","2000-11-30 22:45:00.742"]],"header":["ID","NAME","DOB","DOB2","DOB3","DOB4"]}}]